staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -107,6 +107,15 @@ func MonitorRequest(request *http.Request, verb, resource, subresource, scope, c
 		}
 	}
 
+	// make get pod log with follow option api request reported as watch
+	if verb == "GET" && resource == "pods" && subresource == "log" {
```

Comment:
I'm not quite sure about how many api requests are using URL param keys? If we use it, how can we record them as a label, For example, a request like this(have three param keys)
```
POST someuri?watch=false&follow=true&detail=false
```